core/string/unicode_normalized_spec.rb

```diff
@@ -0,0 +1,105 @@
+require File.expand_path('../../../spec_helper', __FILE__)
```

Comment:
There should be `# encoding: utf-8` since you use unicode string literals after (mostly useful for older versions which are not UTF-8 by default).